### ISSUE TITLE
Curve adjustments from a previous file should not be the default curves for a new file

### ISSUE THREAD

**Issue by SaturnusDJ** [Beep6581/RawTherapee#4575]
https://github.com/Beep6581/RawTherapee/issues/4573

I don't see any benefit to it. Someone does? 
In my case I would never require for example the exact same HSV curves in different photos.

**Comment by Thanatomanic** [comment]
Continued from #4573 

From personal experience I know I am more annoyed than happy in having persistent curves. In the vast majority of cases I expect my images to be a clean slate when I start editing. If I want some parameters to match a series of images, I have (automated) presets.

> Imagine you're working on a series of images, some are ISO100 and some are ISO6400. Very different regarding required sharpening, noise reduction, demosaicing, etc. i.e. you would not copy the PP3 from one to the other because they are too different. But the required curves would look very similar, since curves are independent of ISO.

@Beep6581 Can you give an example of series of closely related images that require similar color modifications, but are varying greatly in ISO value? And would there be other parameters that might vary between images, so that you would want to refrain from having identical sharpening, noise reduction, etc.? If not, then I would argue that having persistent curves has only one use case. Which would not be a strong argument to keep them. But let's discuss :-)

**Comment by Beep6581** [comment]
@Thanatomanic I agree in the lack of _strong_ arguments for. It's a convenience I grew accustomed to and find useful, but still realize it's odd. Let's see what the others think.
